packages/pixelboy/apps/journal/checkin/checkin.coffee

```diff
@@ -47,3 +49,34 @@ class PAA.PixelBoy.Apps.Journal.CheckIn extends AM.Component
 
       else
         FlowRouter.go 'journal'
+
+  getExternalImgUrl: (event) ->
+    url = event.currentTarget.value
+    externalImgUrl = null
+    imgPreview = $('.image-preview')
+    placeholder = 'http://placehold.it/350x150?text=your+art'
+    errorIndicator = $('.preview-error')
+
+    if /twitter\.com/.test(url)
+      id = url.split('/status/')[1]
+      Meteor.call 'getImgFromTweet', id, (error, data) =>
+        if !error
```

Comment:
We don't use `!`, instead use `not`.

In this case you should go one step further. In coffeescript, `if not` would replaced with `unless`.

In fact we go even further. So you don't eventually get nested branching (if inside if inside if …), the conditions which prevent the general flow to execute are handled and then the function returns. So this callback would be rewritten as:
```
if error
  imgPreview.attr('src', placeholder)
  errorIndicator.html('There was an error communicating with the server. Either the tweet doesn\'t exist, or the server is down - try again later!')
  return

externalImgUrl = data.entities.media[0].media_url_https
…
```